experimental/traffic-portal/src/app/api/testing/server.service.ts

```diff
@@ -41,7 +41,7 @@ function serverCheck(server: ResponseServer): Servercheck {
 		cacheGroup: server.cachegroup ?? "SERVER HAD NO CACHE GROUP",
 		hostName: server.hostName ?? "SERVER HAD NO HOST NAME",
 		id: server.id,
-		profile: server.profile ?? "SERVER HAD NO PROFILE",
+		profile: server.profileNames.length > 0 ? server.profileNames[0] :  "SERVER HAD NO PROFILE",
```

Comment:
We don't have `noUncheckedIndexedAccess` enabled, so fwiw you can just do `server.profileNames[0] ?? "SERVER HAD NO PROFILE"`